src/components/Shelf/Shelf.tsx

```diff
@@ -71,22 +71,24 @@ const Shelf = ({
 
   const renderTile = useCallback(
     (item: PlaylistItem, isInView: boolean) => {
+      const url = mediaURL({ media: item, playlistId: playlist.feedid, play: type === PersonalShelf.ContinueWatching });
       return (
-        <Card
-          key={item.mediaid}
-          progress={watchHistory ? watchHistory[item.mediaid] : undefined}
-          onClick={isInView ? () => onCardClick(item, playlist.feedid, type) : undefined}
-          onHover={typeof onCardHover === 'function' ? () => onCardHover(item) : undefined}
-          featured={featured}
-          disabled={!isInView}
-          loading={loading}
-          isLocked={isLocked(accessModel, isLoggedIn, hasSubscription, item)}
-          posterAspect={posterAspect}
-          item={item}
-        />
+        <a href={isInView ? url : undefined} className={styles.cardWrapper}>
```

Comment:
I think it is better to update the `Card` component because now it results in a button wrapped by an anchor tag. This negatively impacts the accessibility for keyboard/screenreader users. Besides that, the Card component is also used on the playlist screens, which use the `CardGrid` component.

**Current HTML**
```html
<a href="/m/uB8aRnu6/agent-327?r=dGSUzs9o" class="_cardWrapper_1ge8j_66">
  <div class="_card_1lln7_1" tabindex="0" role="button" aria-label="Play Agent 327">
```

**Expected HTML**
```html
<a href="/m/uB8aRnu6/agent-327?r=dGSUzs9o" class="_card_1lln7_1" aria-label="Agent 327">
```

I also see that the `aria-label` contains the prefix "Play " which can be removed.